action.yml

```diff
@@ -30,6 +32,41 @@ inputs:
       list.
     required: false
     default: ""
+
```

Comment:
In the interest of consistency, can we order the parameters the same way as in this link?

https://marketplace.visualstudio.com/items?itemName=MathWorks.matlab-azure-devops-extension